certbot/docs/using.rst

```diff
@@ -516,6 +516,51 @@ revocation from any ACME account::
 
   certbot revoke --cert-path /etc/letsencrypt/live/example.com/cert.pem --key-path /etc/letsencrypt/live/example.com/privkey.pem
 
+.. _deleting:
+
+Deleting certificates
+---------------------
+
+If you need to delete a certificate, use the ``delete`` subcommand.
+
+.. note:: Do not manually delete certificate files from inside ``/etc/letsencrypt/``. Always use the ``delete`` subcommand.
+
+A certificate may be deleted by providing its name with ``--cert-name``. \
+You may find its name using ``certbot certificates``.
+
+Otherwise, you will be prompted to choose one or more
+certificates to delete::
+
+  certbot delete --cert-name example.com
+  # or to choose from a list:
+  certbot delete
+
+**Safely deleting certificates**
+
+*Before* deleting a certificate, make sure to remove all references to that certificate from the configuration
```

Comment:
I guarantee people will do the top thing and only after, read this. Some options that come to mind to address that:
- switch the order?
- Put a note that says "read this entire section carefully"

Reply:
Hehehe, true. I've gone for the note.